docs/developer/app-store/apps/search.mdx

```diff
@@ -23,6 +23,7 @@ The application provides integration with [Algolia](https://www.algolia.com/) se
 
 - Initial synchronization has to be done manually in the App
 - The application creates a separate record for each variant
+- A single variant size is limited due to Algolia record size constraint. That means the data attached to your variants such as the number of attributes or wrong field filtering configuration in the app settings can lead to exceeding that limitation. In that case, the app cannot store your record.
```

Comment:
Maybe worth including a link or providing more info what exactly is the limit? Like I know that there is a limit, but I don't know if the limit is related to a single object(variant), or all variants.

Reply:
@korycins "A **single variant** size is limited due to Algolia record size constraint" - but I agree that i can add link to the alolia docs where they explain record limit